utils/funcexp/func_space.cpp

```diff
@@ -29,6 +29,7 @@ namespace funcexp
 CalpontSystemCatalog::ColType Func_space::operationType(FunctionParm& fp,
                                                         CalpontSystemCatalog::ColType& resultType)
 {
+  resultType.colWidth = 32;
```

Comment:
Could you elaborate on the value of the constant?

Reply:
I cannot. Having smaller values may break some tests, from I remember.

It can be that now it is not necessary.

Reply:
I'll look.

Reply:
It is not needed.